pkg/tracingpolicy/mode.go

```diff
@@ -1,26 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
let's not add this file in the first place?

Reply:
fixed, thanks